metricbeat/mb/event.go

```diff
@@ -44,6 +44,8 @@ type Event struct {
 	Host      string        // Host from which the data was collected.
 	Service   string        // Service type
 	Took      time.Duration // Amount of time it took to collect the event data.
+
+	DisableTimeSeries bool // true if the event doesn't contain timeseries data
```

Comment:
How would this field be used? Is it intended to be able to disable time series per metricset?

Reply:
It was introduced to allow metricbeat to continue sending events that are not metrics AFAIK

Reply:
Oh, for events and so on, I see.